docs/src/develop/GetStarted.htm

```diff
@@ -49,6 +49,8 @@
             </li>
             <li>
             	<p>Use the Grommet generator to bootstrap your new app</p>
+              <p>If you get 'Could not find Grommit' error make sure you set the
```

Comment:
Can you make sure that the spacing is write in this file ? I think we have one extra unnecessary space.